src/components/Image.tsx

```diff
@@ -2,11 +2,12 @@ import { useImageEngineContext } from "../context"
 import { constructUrl, generateSrcSetString } from "../utils"
 import { TImageProps } from "../types"
 
-const ALLOWED_INPUT_EXTENSIONS = [
+import { IEFormat, IEFit, IEDirectives } from "@imageengine/imageengine-helpers";
+
+const ALLOWED_INPUT_EXTENSIONS: IEFormat[] = [
```

Comment:
type and removed `jpeg` is there any reason why `jpeg` is there and does it work as a directive?

Reply:
`jpeg` is a valid extension and some people insist in using it after all these years even when the rest of use have used `jpg`.  I have had to add support for `jpeg` many times so please don't break it.  Honestly, I'm not sure why we even check the file extensions since ImageEngine will serve files of any type.

Reply:
Got it. Good to know then I'll place it back and also to the type then.
I'm not sure, I guess the idea is to make sure you're using an image format for which directives can be applied? As in although it will mirror any file, if they aren't processable then you won't be using the directives you placed correct?